code/modules/admin/smites/ghost_control.dm

```diff
@@ -0,0 +1,51 @@
+/datum/smite/ghost_control
+	name = "Ghost Control"
+
+/datum/smite/ghost_control/effect(client/user, mob/living/target)
+	target.AddComponent(/datum/component/deadchat_control/cardinal_movement, ANARCHY_MODE, list(
+		"clap" = CALLBACK(target, TYPE_PROC_REF(/mob, emote), "clap"),
+		"cry" = CALLBACK(target, TYPE_PROC_REF(/mob, emote), "cry"),
+		"flip" = CALLBACK(target, TYPE_PROC_REF(/mob, emote), "flip"),
+		"laugh" = CALLBACK(target, TYPE_PROC_REF(/mob, emote), "laugh"),
+		"scream" = CALLBACK(target, TYPE_PROC_REF(/mob, emote), "scream"),
+		"spin" = CALLBACK(target, TYPE_PROC_REF(/mob, emote), "spin"),
+		"swear" = CALLBACK(target, TYPE_PROC_REF(/mob, emote), "swear"),
+		"drop" = CALLBACK(target, TYPE_PROC_REF(/mob, drop_all_held_items)),
+		"fall" = CALLBACK(target, TYPE_PROC_REF(/mob/living, Knockdown), 1 SECONDS),
+		"stand" = CALLBACK(target, TYPE_PROC_REF(/mob/living, resist_buckle)),
+		"throw" = CALLBACK(target, TYPE_PROC_REF(/mob, throw_item), get_edge_target_turf(target, pick(GLOB.alldirs))),
+		"shove" = CALLBACK(src, PROC_REF(ghost_shove), target),
+		"sit" = CALLBACK(src, PROC_REF(ghost_sit), target),
+		"run" = CALLBACK(src, PROC_REF(ghost_speed), target, MOVE_INTENT_RUN),
+		"walk" = CALLBACK(src, PROC_REF(ghost_speed), target, MOVE_INTENT_WALK),
+		), 7 SECONDS)
```

Comment:
i can alphabetize this better without issue but i liked the list being by the /kind/ of action it is more than being fully alphabetized. it looks nicer to me this way but i get it